eNMS/static/js/services/netconf_service.js

```diff
@@ -0,0 +1,88 @@
+// eslint-disable-next-line
+function job(id) {
+  // timout of 50ms required to wait for form data to load
+  setTimeout(() => {
+    // allocate the dropdown
+    const operationDropdownId = "netconf_service-nc_type";
+    const operationDropdown = getJQueryObject(operationDropdownId, id);
+
+    // maintain state
+    // this is the data passed in through the view
+    let inputData = JSON.parse($("#input_data").val());
+    const fields = inputData["fields"];
+    const netconfType = inputData["netconf_type"];
+
+    // get the current dropdown value and update UI
+    let operation = operationDropdown.val();
+    updateElements(fields, netconfType, operation);
```

Comment:
do you really need that ? I'm not seeing a difference with or without it. Even if you do, this is duplicated code as it is the same thing as inside the change event, so you can trigger it by appending `.trigger("change");`
which means you're only using `updateElements`, so you don't need it to be a function